src/main/scala/com/fulcrumgenomics/util/NumericCounter.scala

```diff
@@ -35,6 +35,13 @@ object NumericCounter {
     ts.foreach(counter.count)
     counter
   }
+
+  /** Generates a counter that has counted all the numerics provided. */
+  def from[T](ts: TraversableOnce[(T, Long)])(implicit numeric: Numeric[T]) : NumericCounter[T] = {
```

Comment:
Let's leave this here for now, but I think ideally this could be replaced/removed it we implement CanBuildFrom.